terraform/modules/common/instance/main.tf

```diff
@@ -0,0 +1,63 @@
+## Variables
+
+variable "stack_name" {
+  description = "The name of the stack within which the resource resides"
+}
+
+variable "ami_id" {
+  description = "The ami which will be used to build the instance"
+}
+
+variable "subnet_id" {
+  description = "The subnet which the instance will be added to"
+}
+
+variable "ssh_key_name" {
+  description = "The name of the ssh key which will assigned to the instance"
+}
+
+variable "security_groups" {
+  description = "A list of security groups which will be added to the instance. If the list is empty the default securit group will be used"
+  type        = "list"
+}
+
+variable "public_ip" {
+  description = "When this is true a public ip is added to the instance"
+  default     = false
```

Comment:
See https://www.terraform.io/docs/configuration/variables.html#booleans as I believe it is suggested to use `"false"`

Reply:
can do, but there are a few places where this is not done in the codebase